crates/swc/src/plugin.rs

```diff
@@ -40,6 +42,7 @@ pub fn plugins(
     comments: Option<swc_common::comments::SingleThreadedComments>,
     source_map: std::sync::Arc<swc_common::SourceMap>,
     unresolved_mark: swc_common::Mark,
+    #[cfg(feature = "plugin")] plugin_runtime: Arc<dyn PluginRuntime>,
```

Comment:
`#[cfg]` should never be placed on a parameter

Reply:
Ok. I noticed this isn't a public API and is only called from one place. so I'll put the function under cfg.